lib/Myriad/Transport/Redis.pm

```diff
@@ -78,6 +80,20 @@ Number of items to allow per batch (pending / readgroup calls).
 
 method batch_count () { $batch_count }
 
+
+#inject the root namespace before every command
+
+install_modifier('Net::Async::Redis::Commands', 
```

Comment:
Can't do this.

It breaks Net::Async::Redis direct usage for any non-framework code - for example, `async method some_rpc : RPC { return await $redis->get('arbitrary.key') }`.